lib/membrane_rtmp_plugin/rtmp/source/bin.ex

```diff
@@ -1,9 +1,10 @@
 defmodule Membrane.RTMP.SourceBin do
   @moduledoc """
-  Bin responsible for spawning new RTMP server.
+  Bin responsible for demuxing and parsing a video stream.
 
-  It will receive RTMP stream from the client, parse it and demux it, outputting single audio and video which are ready for further processing with Membrane Elements.
-  At this moment only AAC and H264 codecs are support
+  The bin requires the RTMP client to be already connected.
+  Outputs single audio and video which are ready for further processing with Membrane Elements.
```

Comment:
The question probably could be if we intend to to support streams that can contains just video or just audio. OBS may always send the two but it won't be always the case anyways.

Reply:
We want to stream just audio or video, and that should be possible now.